apacheconfig/wloader.py

```diff
@@ -31,19 +31,135 @@ def dump(self):
     def typestring(self):
         """Returns object typestring as defined by the apacheconfig parser."""
 
-    @abc.abstractproperty
-    def whitespace(self):
-        """Returns preceding or trailing whitespace for this node as a string.
+
+class ListNode(AbstractASTNode):
+    """Creates object representing an ordered list of LeafASTNodes.
```

Comment:
Would it make sense to make this class fully supporting [Sequence](https://docs.python.org/3/library/stdtypes.html#sequence-types-list-tuple-range) type protocol? That might have a benefit of familiar interface and ability to participate in sequence-handling operators (as operands)...

Can this serve as an illustration of sheer over engineering?

Reply:
Oh, we had this discussion internally too! It might be good in the future to provide this, but for the time being we'd like to explicitly limit the API & surface area to the functions we need (less testing, etc)